jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -114,6 +114,11 @@
 	private static String fileEncoding;
 	private static String osEncoding;
 
+	/*[IF JAVA_SPEC_VERSION >= 18]*/
+	private static final int sysPropID_OSVersion = 0;
+	private static String sysPropOSVersion;
```

Comment:
I think this can be declared final as well, since it's initialized in the static initializer.

Reply:
Added `final` declaration.